<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>780</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>780</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>780</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>红黑夜法官</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/image/titleicon.jpg</normaloff>:/image/titleicon.jpg</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="picturelabel">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>150</y>
      <width>271</width>
      <height>381</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTableWidget" name="tiptable">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>10</y>
      <width>391</width>
      <height>421</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QLabel" name="statuslabel1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>24</y>
      <width>121</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>新宋体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>当前阶段：</string>
    </property>
   </widget>
   <widget class="QLabel" name="statuslabel2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>151</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>新宋体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>白天</string>
    </property>
   </widget>
   <widget class="QLabel" name="statuslabel3">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>30</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>村长选举发言阶段</string>
    </property>
   </widget>
   <widget class="QPushButton" name="speechbtn">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>680</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>发言</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="Input_textEdit">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>580</y>
      <width>331</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>18</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>80</y>
      <width>124</width>
      <height>35</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="playerlabel2">
       <property name="font">
        <font>
         <family>Lato</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 255, 127);</string>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="playerlabel1">
       <property name="font">
        <font>
         <family>新宋体</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 255, 0);</string>
       </property>
       <property name="text">
        <string>号玩家</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>550</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="TimeLabel">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>550</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;黑体&quot;;
color:rgb(255, 255, 127)</string>
    </property>
    <property name="text">
     <string>回合剩余时间</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="Output_textEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>231</width>
      <height>291</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(85, 255, 127);
background-color: rgba(255, 132, 139, 0);</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="endTitleLabel">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>50</y>
      <width>261</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>48</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>游戏结束</string>
    </property>
   </widget>
   <widget class="QLabel" name="endTextLabel">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>240</y>
      <width>311</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QToolButton" name="exitToolButton">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>450</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/exitbutton.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QToolButton" name="restartToolButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>450</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/restartbutton.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="hideButton">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>440</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>04b_21</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>隐藏</string>
    </property>
   </widget>
   <widget class="QPushButton" name="showButton">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>440</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>04b_21</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>显示</string>
    </property>
   </widget>
   <widget class="QLabel" name="tableLabel">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>430</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 170, 255);</string>
    </property>
    <property name="text">
     <string>玩家信息表</string>
    </property>
   </widget>
   <widget class="QLabel" name="speechLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 170, 127);</string>
    </property>
    <property name="text">
     <string>发言内容</string>
    </property>
   </widget>
   <widget class="QLabel" name="daylabel2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>110</y>
      <width>49</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Palatino Linotype</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 255, 255);</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="daylabel1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>104</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>新宋体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 255, 255);</string>
    </property>
    <property name="text">
     <string>天数：</string>
    </property>
   </widget>
   <zorder>picturelabel</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>tiptable</zorder>
   <zorder>statuslabel1</zorder>
   <zorder>statuslabel2</zorder>
   <zorder>statuslabel3</zorder>
   <zorder>speechbtn</zorder>
   <zorder>Input_textEdit</zorder>
   <zorder>lcdNumber</zorder>
   <zorder>TimeLabel</zorder>
   <zorder>Output_textEdit</zorder>
   <zorder>endTitleLabel</zorder>
   <zorder>endTextLabel</zorder>
   <zorder>exitToolButton</zorder>
   <zorder>restartToolButton</zorder>
   <zorder>hideButton</zorder>
   <zorder>showButton</zorder>
   <zorder>tableLabel</zorder>
   <zorder>speechLabel</zorder>
   <zorder>daylabel2</zorder>
   <zorder>daylabel1</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>开始</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
